exdb/urls.py

```diff
@@ -18,13 +18,12 @@
 from exdb import views
 
 urlpatterns = [
-    url(r'^$', views.WelcomeView.as_view(), name='welcome'),
+    url(r'^$', views.HomeView.as_view(), name='home'),
     url(r'^create$', views.CreateExperienceView.as_view(), name='create_experience'),
-    url(r'^hallstaff_dash$', views.HallStaffDashboardView.as_view(), name='hallstaff_dash'),
     url(r'^approval/(?P<pk>\d+)$', views.ExperienceApprovalView.as_view(), name='approval'),
-    url(r'^home$', views.RAHomeView.as_view(), name='ra_home'),
     url(r'^conclusion/(?P<pk>\d+)$', views.ExperienceConclusionView.as_view(), name='conclusion'),
     url(r'^view/experience/(?P<pk>\d+)$', views.ViewExperienceView.as_view(), name='view_experience'),
     url(r'^view/edit/(?P<pk>\d+)$', views.EditExperienceView.as_view(), name='edit'),
     url(r'^login$', login, name='login', kwargs={'template_name': 'exdb/login.html'}),
+    url('^logout$', views.LogoutView.as_view(), name='logout'),
```

Comment:
Why not use [Django's built-in logout view](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.views.logout)? Then we won't need tests for it either.